contents/data_vis_makie_colors.md

```diff
@@ -97,13 +102,23 @@ scat = """
     """
 sco(scat)
 ```
+<!--
+Sorry, but I don't like this example, because the same could be achieved much more easily.
+- `cmap` is not used -> either add a comment or text what this *could* be good/used for or delete it.
+- Either state 3 colors in in the `ColorScheme` and omit the `values` parameter of `cgrad` OR
+  state 2 colors in the `ColorScheme` and use the `values` parameter of `cgrad`.
+  Using both is redundant.
```

Comment:
in cgrad you always need to indicate how many colors you want. So, the goal here is to give the user several options to do a Colorbar and see what do you actually need. It seem to have worked as you already noticed. Good idea regarding bad, neutral and good I will take this into account later.

Reply:
That doesn't seem to be right, because

```jl
using Colors, ColorSchemes
using CairoMakie
CairoMakie.activate!()

figure = (; resolution=(600, 400), font="CMU Serif")
axis = (; xlabel=L"x", ylabel=L"y", aspect=DataAspect())
mygrays = ColorScheme([Gray(i) for i in [0.0, 0.5, 1.0]])
fig, _, pltobj = heatmap(
    rand(-1:1, 20, 20);
    colormap=cgrad(mygrays, categorical=true, rev=true),
    axis=axis,
    figure=figure,
)
cbar = Colorbar(fig[1, 2], pltobj, label="Categories")
cbar.ticks = ([-0.66, 0, 0.66], ["bad", "neutral", "good"])
fig
```

works perfectly fine for me (in Julia v1.7). This is supported by the output of `?cgrad()` which gives

```jl
cgrad(colors, [values]; categorical = nothing, scale = nothing, rev = false, alpha = nothing)
```

and I guess the arguments in square brackets are optional.